pylabrobot/resources/revvity/plates.py

```diff
@@ -0,0 +1,43 @@
+""" Revvity plates """
+
+# pylint: disable=invalid-name
+
+from pylabrobot.resources.plate import Plate
+from pylabrobot.resources.well import Well, WellBottomType
+from pylabrobot.resources.itemized_resource import create_equally_spaced
+
+
+# TODO: update heights for volume calculations
```

Comment:
Is the current implementation reasonably accurate? Otherwise it might be best to throw a NotImplementedError imo

Reply:
The plate definition is reasonably accurate, and I've used it quite a bit but the volume calculation is not yet. So I changed the volume calculation method to raise a NotImplementedError as you suggested. Thank you for checking.

Reply:
Makes sense!